internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -108,7 +108,29 @@ func serverCreateCommand() *core.Command {
 			Short:   "List marketplace label images",
 			Command: "scw marketplace image list",
 		}},
-		Examples: []*core.Example{}, // TODO: Add examples [APIGW-1371]
+		Examples: []*core.Example{
+			{
+				Short:   "Create and start an instance on Ubuntu Bionic",
+				Request: `{"image":"ubuntu_bionic","start":true}`,
+			},
+			{
+				Short:   "Create a GP1-XS instance, give it a name and add tags",
+				Request: `{"image":"ubuntu_bionic","type":"GP1-XS","name":"foo","tags":["prod","blue"]}`,
+			},
+			{
+				Short:   "Create an instance with 2 additional block volumes (50GB and 100GB)",
```

Comment:
We could add `and a default root volume`

Reply:
It sounds confusing to write this because it is the default behavior for all the above commands as well.

Reply:
I can't see where it is documented for our users. Maybe it is the good PR to fille the long usage (cf l.42)

Reply:
Yes, all custom instance commands deserve a second pass on the documentation, but this is not the scope of this PR